webhook-connector/order-events.md

```diff
@@ -0,0 +1,131 @@
+
+## Order confirmed by supplier or buyer
+
+The supplier can either accept, reject or propose an alternative (see next section).
+
+* Accepted means that the supplier confirmed the order line(s) as requested by the buyer.
+* Rejecting means that the supplier cannot deliver the order line(s) at all.
+* Confirmed by supplier happens when the supplier confirms and [Auto Confirm](https://docs.tradecloud1.com/api/processes/order/buyer/issue/indicators#auto-confirm) has been enabled by the buyer.
+* Confirmed by buyer is used when the buyer migrates already confirmed order lines to Tradecloud.
+
+| OrderEvent                      | Webhook Configuration                     |
+| ------------------------------- | ----------------------------------------- |
+| `OrderLinesAcceptedBySupplier`  | Order lines are accepted by the supplier  |
+| `OrderLinesRejectedBySupplier`  | Order lines are rejected by the supplier  |
+| `OrderLinesConfirmedBySupplier` | Order lines are confirmed by the supplier |
+| `OrderLinesConfirmedByBuyer`    | Order lines are confirmed by the buyer    |
+
+
+## Order proposal by supplier
+
+The supplier can propose an alternative delivery schedule or prices.
+
+The buyer either approves or rejects the proposal.
+
+| OrderEvent                            | Webhook Configuration                                        |
+| --------------------------------------| ------------------------------------------------------------ |
+| `OrderChangesProposedBySupplier`      | Order changes are proposed by the supplier                   |
+| `OrderChangesProposalApprovedByBuyer` | By supplier proposed order changes are approved by the buyer |
+| `OrderChangesProposalRejectedByBuyer` | By supplier proposed order changes are rejected by the buyer |
+
+## Order updated by supplier
+
+The supplier can update existing order lines.
+
+If delivery schedule or prices are updated this will become either a proposal or reopen request.
+
+| OrderEvent                     | Webhook Configuration                        |
+| ------------------------------ | -------------------------------------------- |
+| `OrderLinesUpdatedBySupplier`  | Order and line fields other than prices & delivery schedules are updated by the supplier |
+| `OrderLinesItemDetailsChanged` | Order lines item details are changed by the supplier | 
+
+## Order reopen request by buyer
+
+The buyer can request to reopen confirmed order lines.
+
+The supplier can either approve or reject the reopen request.
+
+| OrderEvent                                  | Webhook Configuration                        |
+| ------------------------------------------- | -------------------------------------------- |
+| `OrderLinesReopenRequestedByBuyer`          | Order lines reopen is requested by buyer     |
+| `OrderLinesReopenRequestApprovedBySupplier` | Buyer reopen request is approved by supplier |
+| `OrderLinesReopenRequestRejectedBySupplier` | Buyer reopen request is rejected by supplier |
```

Comment:
```suggestion
| `OrderLinesReopenRequestedByBuyer`          | Order lines reopen is requested by the buyer     |
| `OrderLinesReopenRequestApprovedBySupplier` | Buyer reopen request is approved by the supplier |
| `OrderLinesReopenRequestRejectedBySupplier` | Buyer reopen request is rejected by the supplier |
```

Reply:
The text in the table are the literal texts in the webhook configuration you may choose.